### PR TITLE
Add brotli as a project

### PR DESCRIPTION
**Pull request by Sinerum**
adds [brotli](https://github.com/google/brotli) as a project.

### CODE REVIEW

**Review by boehmseb — APPROVED** [review]
lgtm

File: varats/varats/projects/c_projects/brotli.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    """Brotli compression format."""
+
+    NAME = 'brotli'
+    GROUP = 'c_projects'
+    DOMAIN = 'compression'
+
+    SOURCE = [
+        bb.source.Git(
+            remote="https://github.com/google/brotli.git",
+            local="brot",
```
Comment (vulder) [inline varats/varats/projects/c_projects/brotli.py]: So, if I understand correctly the bug was only related to the local folder name?

Maybe we should switch this to brotli_repo or something like that, as it's more descriptive.

File: varats/varats/projects/c_projects/brotli.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        """Compile the project."""
+        brotli_version_source = local.path(self.source_of_primary)
+
+        c_compiler = bb.compiler.cc(self)  # type: ignore
+        mkdir(brotli_version_source / "out")
+        with local.cwd(brotli_version_source / "out"):
+            with local.env(CC=str(c_compiler)):
+                bb.watch(local["../configure-cmake"])()
+            bb.watch(make)("-j", get_number_of_jobs(bb_cfg()))
+        with local.cwd(brotli_version_source):
```
Comment (vulder) [inline varats/varats/projects/c_projects/brotli.py]:
```suggestion
            bb.watch(make)("-j", get_number_of_jobs(bb_cfg()))

        with local.cwd(brotli_version_source):
```

**Review by vulder — APPROVED** [review]

File: varats/varats/projects/c_projects/brotli.py (hunk 1 above)
Comment (vulder) [reply varats/varats/projects/c_projects/brotli.py]:
```suggestion
            local="brotli_git",
```

File: varats/varats/projects/c_projects/brotli.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+from varats.containers.containers import get_base_image, ImageBase
+from varats.paper_mgmt.paper_config import project_filter_generator
+from varats.project.project_util import (
+    ProjectBinaryWrapper,
+    get_all_revisions_between,
+    wrap_paths_to_binaries,
+    get_local_project_git_path,
+    BinaryType,
+    verify_binaries,
```
Comment (vulder) [inline varats/varats/projects/c_projects/brotli.py]:
```suggestion
    ProjectBinaryWrapper,
    wrap_paths_to_binaries,
    BinaryType,
    verify_binaries,
```

File: varats/varats/projects/c_projects/brotli.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
+"""Project file for gravity."""
+import typing as tp
+
+import benchbuild as bb
+from benchbuild.utils.cmd import cmake, mkdir, make
```
Comment (vulder) [inline varats/varats/projects/c_projects/brotli.py]:
```suggestion
from benchbuild.utils.cmd import mkdir, make
```

**Pull request merged**